internal/jimm/watcher.go

```diff
@@ -498,6 +493,7 @@ func (w *Watcher) handleDelta(ctx context.Context, modelIDf func(string) *modelS
 			state.changed = true
 			state.units[eid.Id] = true
 		}
+	default:
```

Comment:
Default and do what?

Reply:
nothing.. ignore deltas we don't care about